sd_bandits/experiment.py

```diff
@@ -212,13 +219,21 @@ def learn_and_obtain_policy_feedback(self):
 
         """
         logging.info("Learning and obtaining policy feedback")
-        for i, policy in enumerate(self.policies):
+        for i, (policy, policy_meta) in enumerate(self.policies):
```

Comment:
`policy_meta` is just optional kwargs to pass to `obtain_bandit_batch_feedback`?